### PR TITLE
Customize Algernon Position, Implement #11

### PR DESCRIPTION
**Pull request by BenBals**

### CODE REVIEW

**Comment by BenBals** [comment]
@sinnaj-r Here the same tests are failing. What do we need to do to make them pass?

**Comment by sinnaj-r** [comment]
> @sinnaj-r Here the same tests are failing. What do we need to do to make them pass?

You should be fine just increasing the timeout:

https://github.com/hpi-swa-teaching/Algernon-Launcher/blob/b90f8b970640108fcca51a9db1a00e821d0669c7/packages/AlgernonTests.package/ALGLinting.class/instance/defaultTimeout.st

**Comment by BenBals** [comment]
I see you did that in #86. To avoid merge conflicts we should merge that pr first and then pull from development to this branch. What do you think? Should we merge #86 now?

**Comment by sinnaj-r** [comment]
> I see you did that in #86. To avoid merge conflicts we should merge that pr first and then pull from development to this branch. What do you think? Should we merge #86 now?

I just merged it 🙂

File: packages/Algernon.package/ALGBreadcrumbs.class/instance/initialize.st [hunk 1]
```diff
@@ -6,6 +6,6 @@ initialize
 		height: self class defaultHeight;
 		width: self controller width;
 		color: self class defaultColor;
-		position: self controller position x @ (self controller position y - self height);
-		lock: true.
-	self initializeDefaultCrumb
\ No newline at end of file
+		position: self controller position x @ (self controller position y - self height).
+		"lock: true."
```
Comment (sinnaj-r) [inline packages/Algernon.package/ALGBreadcrumbs.class/instance/initialize.st]: This comment should be removed or extended by and explanation

File: packages/Algernon.package/ALGCore.class/instance/grabFocus..st [hunk 2]
```diff
@@ -2,5 +2,5 @@ event handling
 grabFocus: hand
 
 	hand 
-		newKeyboardFocus: self textField;
-		newMouseFocus: self
\ No newline at end of file
+		newKeyboardFocus: self textField
+		"newMouseFocus: self"
```
Comment (sinnaj-r) [inline packages/Algernon.package/ALGCore.class/instance/grabFocus..st]: Another Comment

**Comment by BenBals** [comment]
Nice, I did the merge. Let's see if that helps.

**Review by sinnaj-r — APPROVED** [review]
Apart from the two (probably forgotten) comments, both the code and the feature look good 👍

**Comment by BenBals** [comment]
I have fried my image again, could you remove the comments please, @kej-jay?

File: packages/AlgernonTests.package/ALGTestBackground.class/instance/testBackgroundIsAbandonedCorrectly.st [hunk 3]
```diff
@@ -0,0 +1,7 @@
+testing
+testBackgroundIsAbandonedCorrectly
```
Comment (sinnaj-r) [inline packages/AlgernonTests.package/ALGTestBackground.class/instance/testBackgroundIsAbandonedCorrectly.st:2]: Should we maybe move this test in an ALGTest Subclass ? I find the manual creation and cleanup of Algernon instances a bit "unpleasant".

**Review by sinnaj-r — APPROVED** [review]

File: packages/AlgernonTests.package/ALGTestBackground.class/instance/testBackgroundIsAbandonedCorrectly.st (hunk 3 above)
Comment (BenBals) [reply packages/AlgernonTests.package/ALGTestBackground.class/instance/testBackgroundIsAbandonedCorrectly.st:2]: I don't think we should as this is a destructive use of the core. That would make reuse impossible.

**Pull request merged**